src/server/scripts/Spells/spell_item.cpp

```diff
@@ -4385,6 +4385,67 @@ class spell_item_goblin_bomb : public SpellScriptLoader
     }
 };
 
+enum LinkenBoomerang
+{
+    SPELL_DISARM = 15752,
+    SPELL_STUN   = 15753
+};
+
+class spell_item_linken_boomerang : public SpellScriptLoader
+{
+public:
+    spell_item_linken_boomerang() : SpellScriptLoader("spell_item_linken_boomerang") {}
+
+    class spell_item_linken_boomerang_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_item_linken_boomerang_SpellScript)
+
+            void OnEffectHitTargetDisarm(SpellEffIndex effIndex)
+        {
+            PreventHitDefaultEffect(effIndex);
+        }
+
+        void OnEffectHitTargetStun(SpellEffIndex effIndex)
+        {
+            PreventHitDefaultEffect(effIndex);
+        }
+
+        void OnEffectLaunchTargetDisarm(SpellEffIndex effIndex)
+        {
+            PreventHitDefaultEffect(effIndex);
+
+            if (roll_chance_i(3)) // 3% from wiki n shit
```

Comment:
Could it be an idea to move the two magic numbers into a [symbolic constant](https://github.com/azerothcore/wiki/blob/f7eab25947d3e1f8164d8091596dab0214dd9c34/docs/cpp-code-standards.md#random-numbers-vs-constants)?

Reply:
And perhaps keep the language formal